.travis.yml

```diff
@@ -1,25 +1,33 @@
 # This provides us with GCC binary support so we can run bash
+dist: trusty
 language: generic
 
 sudo: required
 
 script:
   - ./build.sh
-  - 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then bash ./scripts/aptRepoUpdater.sh; fi'
+
+notifications:
+email:
+recipients:
+  - xeon.harsh@gmail.com
+  - $email
```

Comment:
Is Travis sends mail through environment variable, is so then ok,
otherwise, remove this.